bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -145,6 +145,16 @@ default Boolean isEnabled() {
         return true;
     }
 
+    /**
+     * Returns the hamburger menu support
+     *
+     * @return the hamburger menu support of the form
+     * @since com.adobe.cq.forms.core.components.models.form 4.5.0
+     */
+    default Boolean getHamburgerMenu() {
+        return false;
```

Comment:
Please add test cases for all files reported by codecov

Reply:
Added all.